nixos/modules/services/monitoring/vmagent.nix

```diff
@@ -1,18 +1,78 @@
-{ config, pkgs, lib, ... }:
-
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
```

Comment:
Please don't mix formatting with functional changes.
This only makes it harder to review and if we ever have to revert (and later reapply) the change, it's also gonna cause unnecessary merge conflicts.

Reply:
The formatting was done as per the `nixfmt-check` CI requirement. I agree that it has made my content updates more cumbersome, have reverted all the formatted content back to its original state.

Reply:
There were a bunch of unrelated changes. My general suggestion is: make one commit that does formatting and another one after that making actual changes. Then, reviewers can go through the changes commit-by-commit.